package.json

```diff
@@ -36,8 +36,16 @@
 		"eslint": "^6.4.0",
 		"eslint-config-prettier": "^6.3.0",
 		"eslint-plugin-import": "^2.18.2",
-		"mocha": "^6.2.0",
-		"nyc": "^14.1.1",
+		"jest": "^24.9.0",
 		"prettier": "1.18.2"
+	},
+	"jest": {
+		"verbose": true,
+		"collectCoverage": true,
+		"expand": true,
+		"roots": [
```

Comment:
The `roots` key seems unnecessary too. Removing it does not impact the tests, as far as I can tell.